packages/cli/src/index.ts

```diff
@@ -410,6 +412,36 @@ yargs
       })(argv);
     }
   )
+  .command(
+    'preview',
+    'Preview Redocly project',
+    (yargs) =>
+      yargs.options({
+        product: {
+          type: 'string',
+          choices: ['redoc', 'revel', 'reef', 'realm', 'redoc-revel', 'redoc-reef', 'revel-reef'],
+        },
+        plan: {
+          type: 'string',
+          choices: PRODUCT_PLANS,
+          default: 'enterprise',
+          description: '',
+        },
+        port: {
+          alias: 'p',
+          type: 'number',
+          description: 'Preview port.',
+        },
+        'project-dir': {
```

Comment:
Does it make sense to point to a specific file instead of a directory for `redoc`?

Reply:
This is not the open source Redoc and this one can only accept a directory.